terraform/modules/gro_ingestion_service/lambdas/src/functions/publishEvent.ts

```diff
@@ -39,7 +43,7 @@ const makeRequest = async (url: string, options: RequestOptions, requestData: st
     })
 
 const getAccessToken = async () => {
-    const authRequest = JSON.stringify({
```

Comment:
why do we need to do this?

Reply:
The auth request has to be a url encoded form, which means it needs to be querystring and not JSON